mozilla/bedrock
# Remove accounts popup from download flow (Fix #14246)

## One-line summary

Remove accounts popup from download flow.

## Significant changes and points to review

- Remove JS, CSS, and associated HTML
- Remove associated tests
- Remove a few other unused strings that use the "Join Firefox" language

## Issue / Bugzilla link

Fix #14246 

## Testing

1. Using Firefox but don't be signed into a Mozilla account
2. Try to download from /new
3. Get a download (instead of the popup)

- Make sure I didn't break downloads 😬 
- There was CSS on the page for a second modal (other-platforms) but I can't find the HTML that goes with that so I removed it but I'm worried I am missing something but I really can't find it.

[✅ Integration tests.](https://github.com/mozilla-sre-deploy/deploy-bedrock/actions/runs/8151612245)
